music/package.json

```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "@tensorflow/tfjs-core": "^0.8.5",
     "midiconvert": "^0.4.4",
+    "node": "^10.0.0",
```

Comment:
what is this needed for? if it's just for type definitions, it could be a dev dependency.

Reply:
Done.